Code/src/main/java/nl/utwente/viskell/ghcj/GhciEvaluator.java

```diff
@@ -20,7 +20,7 @@ public GhciEvaluator() throws HaskellException {
 
     @Override
     protected List<String> getModules() {
-        return ImmutableList.of("Data.List", "Data.Maybe", "Data.Either", "Data.Monoid", "Control.Applicative");
+        return ImmutableList.of("Data.List", "Data.Maybe", "Data.Either", "Data.Monoid", "Control.Applicative", "Test.QuickCheck");
```

Comment:
QuickCheck is very common Haskell library, and if someone doesn't have it they can just install it using cabal or stack.